### DISCUSSION TITLE
Connecting Postal to external MySql using Certificate

### DISCUSSION THREAD

**Discussion by joshcadman-easyreply** [postalserver/postal#2990] [Help with using Postal]
I have recently moved my postal DB to Azure to allow for HA and monitoring, but got to an issue where I cannot connect to the database due to it requiring a certificate (BaltimoreCyberTrustRoot.crt.pem).

Has anyone done anything similar with connecting the database using a certificate and knows how it should be configured in the config file?

**Comment by willpower232** [comment]
I don't think it is part of the new config file schema so that appears to be out.

It might be possible with the original config file layout however I cannot for the life of me find any documentation on how to connect Ruby on Rails to a database with SSL. I've seen a few hints but nothing actually concrete so it might be worth asking around in the Postal discord in case you can attract someone who knows more about how the framework works.
